cumulus_library/cli_parser.py

```diff
@@ -15,6 +15,18 @@ def add_target_argument(parser: argparse.ArgumentParser) -> None:
     )
 
 
+def add_table_builder_argument(parser: argparse.ArgumentParser) -> None:
+    """Adds --table_builder arg to a subparser"""
+    parser.add_argument(
+        "--table-builder",
+        help=(
+            "Within a study, specify a specific table builder to run, skipping all "
+            "other files. Primarily intended for development/debugging. Will still "
+            "remove study tables."
```

Comment:
nit: consider using `help=argparse.SUPPRESS` instead and just hiding this if you think this arg is more dangerous than helpful for n00bs.

If you keep it, is a "table builder" anything a researcher would know? I know that's the internal name for it, but this might be a good point to consider if there's a better end-user terminology. (it's a fine name, I have no complaints, but just my brain fired this question off when exposing something for the first time)